PhoneAuthUI/Sources/PhoneLogin/Screen/PhoneAuthScreen.swift

```diff
@@ -0,0 +1,206 @@
+    }
+}
+
+extension PhoneAuthScreenViewModel: PhoneAuthDelegate {
+    func phoneAuth(_ phoneAuth: SnabblePhoneAuth.PhoneAuth, didReceiveAppUser appUser: SnabbleUser.AppUser) {
+        AppUser.set(appUser, forConfig: phoneAuth.configuration)
+    }
+}
+
+public enum PhoneAuthViewKind {
+    case initial
+    case management
+}
+
+public struct PhoneAuthScreen<Header: View, Footer: View>: View {
+    @Environment(\.dismiss) var dismiss
+    
+    let viewModel: PhoneAuthScreenViewModel
+    let viewKind: PhoneAuthViewKind
+    let header: (() -> Header)?
+    let footer: (() -> Footer)?
+
+    @State var phoneNumber: String = "" {
+        didSet {
+            showOTPInput = !phoneNumber.isEmpty
+        }
+    }
+    
+    var onCompletion: ((SnabbleUser.AppUser?) -> Void)?
+    
+    @State var showProgress: Bool = false
+    @State var errorMessage: String = ""
+    
+    @State private var showOTPInput: Bool = false
+
+    public init(phoneAuth: PhoneAuth,
+                kind: PhoneAuthViewKind,
+                header: (() -> Header)?,
+                footer: (() -> Footer)?,
+                onCompletion: ((SnabbleUser.AppUser?) -> Void)? = nil
+    ) {
+        self.viewModel = PhoneAuthScreenViewModel(phoneAuth: phoneAuth)
+        self.viewKind = kind
+        self.header = header
+        self.footer = footer
+        self.onCompletion = onCompletion
+    }
+    
+    @ViewBuilder
+    var numberView: some View {
+        NumberView(
+            kind: viewKind,
+            showProgress: $showProgress,
+            footerMessage: $errorMessage,
+            header: header,
+            footer: footer
+        ) { phoneNumber in
+            sendPhoneNumber(phoneNumber)
+        }
+    }
+    
+    @ViewBuilder
+    var codeView: some View {
+        CodeView(
+            kind: viewKind,
+            phoneNumber: $phoneNumber,
+            showProgress: $showProgress,
+            footerMessage: $errorMessage,
+            verifyCode: { code, phoneNumber in
+                sendOTP(code, phoneNumber: phoneNumber)
+            },
+            rerequestCode: { phoneNumber in
+                sendPhoneNumber(phoneNumber)
+            }
+        )
+    }
+    
+    public var body: some View {
+        NavigationStack {
+            numberView
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationDestination(isPresented: $showOTPInput, destination: {
+                codeView
+            })
+        }
+    }
+    
+    private func startLoading() {
+        errorMessage = ""
+        showProgress = true
+    }
+    
+    private func messageFor(error: Error) -> String {
+        guard case let HTTPError.invalid(_, clientError) = error, let clientError else {
+            return Asset.localizedString(forKey: "Account.genericError")
+        }
+        let message: String
+        switch clientError.type {
+        case "invalid_input":
+            message = "Account.SignIn.error"
```

Comment:
Ich nehme hier nur ein Beispiel, aber es geht um die gesamten `Account.*` Keys.
Die Keys sind nun Bestandteil vom SDK und müssen somit dem Lokalise Projekt hinzugefügt werden. Zusätzlich sollten wir den Prefix `Snabble` hinzufügen.

Ansonsten bricht es sobald wir es in anderen Apps integrieren.

Reply:
Habe ich mit b306f639d1bd59ab604d026e2274f38c49de3395 angepasst. Bitte prüfe es auch nochmal @utilem